showcase/src/app/demo-ui/pages/date-range-picker/demo-date-range-picker-page.component.html

```diff
@@ -1,80 +1,80 @@
 <h1 class="mat-display-3" translate>SHOWCASE.DEMO.DATE_RANGE_PICKER.TITLE</h1>
 <section class="stark-section">
 	<h1 translate>SHOWCASE.DEMO.SHARED.EXAMPLE_VIEWER_LIST</h1>
-	<example-viewer [extensions]="['HTML', 'TS']" filesPath="date-range-picker/full" exampleTitle="SHOWCASE.DEMO.DATE_RANGE_PICKER.TITLE">
-		<div>
-			<stark-date-range-picker
-				rangePickerId="id-range-picker"
-				rangePickerName="name-range-picker"
-				[isDisabled]="disabled"
-				[dateFilter]="customDateFilter"
-				[startDate]="startDate"
-				startDateLabel="STARK.DATE_RANGE_PICKER.FROM"
-				[endDate]="endDate"
-				endDateLabel="STARK.DATE_RANGE_PICKER.TO"
-				[startMinDate]="minDate"
-				[startMaxDate]="maxDate"
-				[endMinDate]="startDate || minDate"
-				[endMaxDate]="maxDate"
-				(dateRangeChanged)="onDateChanged($event)"
-			>
-			</stark-date-range-picker>
-		</div>
-		<mat-checkbox [(ngModel)]="disabled">isDisabled</mat-checkbox>
+
+	<example-viewer
+		[extensions]="['HTML', 'TS']"
+		filesPath="date-range-picker/model"
+		exampleTitle="SHOWCASE.DEMO.DATE_RANGE_PICKER.WITH_MODEL"
+	>
+		<stark-date-range-picker
+			[(ngModel)]="dateRangeModel"
+			(dateRangeChanged)="onDateModelChange()"
+			dateFilter="OnlyWeekdays"
+			[startMinDate]="today"
+			[startMaxDate]="inOneMonth"
+			[endMinDate]="today"
+			[endMaxDate]="inOneMonth"
+			[disabled]="modelDisabled"
+			[required]="true"
+			#dateRangeComponent
+		>
+			<ng-container start-date-errors
+						  *ngFor="let error of getErrorMessages(dateRangeComponent.startDateFormControl)">
+				{{ error | translate }}
+				<br />
+			</ng-container>
+
+			<ng-container end-date-errors *ngFor="let error of getErrorMessages(dateRangeComponent.endDateFormControl)">
+				{{ error | translate }}
+				<br />
+			</ng-container>
+		</stark-date-range-picker>
+		<br /><br />
+		<mat-checkbox [(ngModel)]="modelDisabled">
+			{{ (modelDisabled ? "SHOWCASE.COMMON.DISABLED" : "SHOWCASE.COMMON.ENABLED") | translate }}
+		</mat-checkbox>
+	</example-viewer>
+
+	<example-viewer
+		[extensions]="['HTML', 'TS']"
+		filesPath="date-range-picker/form-group"
+		exampleTitle="SHOWCASE.DEMO.DATE_RANGE_PICKER.WITH_FORM_GROUP"
+	>
+		<stark-date-range-picker [formGroup]="dateRangeFormGroup">
```

Comment:
The `required` attribute is missing in this example...  because the formControls do have the `Validators.required`.

The corresponding example file should be changed as well 😉

Reply:
Do you mean in `showcase/src/assets/examples/date-range-picker/form-group.ts`? because the required is still used in this example. The validation just happens in the demo, not the component.

Or maybe this is to confusing and it should be removed. 🤔